README.md

```diff
@@ -52,3 +52,25 @@ On top of the `atoti` package, it comes with:
   ```bash
   poetry run pylint app/ tests/
   ```
+
+### Deploying on Heroku
+
+The application can be deployed very easily on Heroku. Since we use poetry, you will need to add the following build packs using the CLI:
+
+```bash
+heroku buildpacks:clear
+heroku buildpacks:add https://github.com/moneymeets/python-poetry-buildpack.git
+heroku buildpacks:add heroku/python
+```
+
+Heroku assigns the port that the application needs to bind to in the `PORT` environment variable.
```

Comment:
It works out of the box with the code in this repo so I think we can skip this. Besides, there is already a code comment for that.

```suggestion
```